modules/node-observability-create-custom-resource.adoc

```diff
@@ -10,7 +10,7 @@ You must create and run the `NodeObservability` custom resource (CR) before you
 
 [IMPORTANT]
 ====
-The worker nodes matching the `nodeSelector` specified in `NodeObservability` CR are rebooted. It might take 10 or more minutes to complete.
+If the CRI-O profiling is not enabled on the worker nodes, the `NodeObservabilityMachineConfig` resource gets created and the worker nodes matching the `nodeSelector` specified in `NodeObservability` CR get rebooted. This might take 10 or more minutes to complete.
```

Comment:
Just a suggestion to break that first sentence into two for clarity and tense.
```suggestion
If CRI-O profiling is not enabled on the worker nodes, the `NodeObservabilityMachineConfig` resource gets created. Worker nodes matching the `nodeSelector` specified in `NodeObservability` CR will reboot. This might take 10 or more minutes to complete.
```

Reply:
Addressed